AppBuilder/platform/RowUpdater.js

```diff
@@ -199,40 +231,54 @@ export default class RowUpdater extends ClassUI {
 
       let $form = $$(this.ids.form);
       if ($form) {
-         $form.getChildViews().forEach(($viewCond) => {
+         $form.getChildViews().forEach(($viewContainer) => {
+            let $viewItem = $viewContainer.getChildViews()[0];
+
             // Ignore "Add new" button
-            if (!$viewCond || !$viewCond.$$) return;
+            if (!$viewItem || !$viewItem.$$) return;
 
-            let $fieldElem = $viewCond.$$(this.ids.field);
+            let $fieldElem = $viewItem.$$(this.ids.field);
             if (!$fieldElem) return;
 
             let fieldId = $fieldElem.getValue();
             if (!fieldId) return;
 
-            let $valueElem = $viewCond.getChildViews()[3];
-            if (!$valueElem) return;
+            let $customValueElem = $viewItem.getChildViews()[3];
+            let $processValueElem = $viewItem.getChildViews()[4];
+            if (!$customValueElem && !$processValueElem) return;
 
             let fieldInfo = this._Object.fieldByID(fieldId);
 
-            let val;
-            if (fieldInfo.key == "date" || fieldInfo.key == "datetime") {
-               let currDateCheckbox = $valueElem.getChildViews()[0];
-               if (currDateCheckbox.getValue() == true) {
-                  val = "ab-current-date";
-               } else {
-                  let datePicker = $valueElem.getChildViews()[1];
-                  val = fieldInfo.getValue(datePicker);
+            let val = {
+               fieldId: fieldId
+            };
+
+            // Custom value
+            if ($customValueElem && $customValueElem.isVisible()) {
+               if (
+                  fieldInfo.key == "date" ||
+                  fieldInfo.key == "datetime"
+               ) {
+                  let currDateCheckbox = $customValueElem.getChildViews()[0];
+                  if (currDateCheckbox.getValue() == true) {
+                     val.value = "ab-current-date";
+                  } else {
+                     // Get value from data field manager
+                     val.value = fieldInfo.getValue($customValueElem);
+                  }
+               }
+               // Process value
+               else if ($processValueElem && $processValueElem.isVisible()) {
+                  val.isProcessValue = true;
+                  val.value = $processValueElem.getValue();
                }
             } else {
                // Get value from data field manager
                val = fieldInfo.getValue($valueElem);
             }
 
             // Add to output
-            result.push({
-               fieldId: fieldId,
-               value: val,
-            });
+            result.push(val);
```

Comment:
Changing the data being returned should also cause an update to the Method comments, right?